src/commands/staking/delegate/stake.rs

```diff
@@ -0,0 +1,88 @@
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = super::DelegateStakeContext)]
+#[interactive_clap(output_context = StakeContext)]
+pub struct Stake {
+    /// Enter the amount to stake from the inner account of the predecessor (example: 10NEAR or 0.5near or 10000yoctonear):
+    amount: crate::common::NearBalance,
+    #[interactive_clap(skip_default_input_arg)]
+    /// What is the signer account ID?
+    signer_account_id: crate::types::account_id::AccountId,
+    #[interactive_clap(named_arg)]
+    /// Select network
+    network_config: crate::network_for_transaction::NetworkForTransactionArgs,
+}
+
+#[derive(Clone)]
+pub struct StakeContext(crate::commands::ActionContext);
+
+impl StakeContext {
+    pub fn from_previous_context(
+        previous_context: super::DelegateStakeContext,
+        scope: &<Stake as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        let signer = scope.signer_account_id.clone();
+        let signer_id: near_primitives::types::AccountId = scope.signer_account_id.clone().into();
+        let validator_account_id = previous_context.validator_account_id.clone();
+        let interacting_with_account_ids = vec![validator_account_id.clone()];
+        let amount = scope.amount.clone();
+
+        let on_after_getting_network_callback: crate::commands::OnAfterGettingNetworkCallback =
+            std::sync::Arc::new(move |_network_config| {
+                Ok(crate::commands::PrepopulatedTransaction {
+                    signer_id: signer_id.clone(),
+                    receiver_id: previous_context.validator_account_id.clone(),
+                    actions: vec![near_primitives::transaction::Action::FunctionCall(
+                        near_primitives::transaction::FunctionCallAction {
+                            method_name: "stake".to_string(),
+                            args: serde_json::json!({
+                                "amount": amount.clone().to_yoctonear().to_string()
+                            })
+                            .to_string()
+                            .into_bytes(),
```

Comment:
Use `serde_json::to_vec()` instead of `.to_string().into_bytes()`

Reply:
Done